ios/Classes/AblyFlutterPlugin.m

```diff
@@ -39,6 +39,44 @@ -(nullable AblyFlutter *)ablyWithHandle:(NSNumber *)handle;
     [plugin registerWithCompletionHandler:result];
 };
 
+static FlutterHandler _createRestWithOptions = ^void(AblyFlutterPlugin *const plugin, FlutterMethodCall *const call, const FlutterResult result) {
+    AblyFlutterMessage *const message = call.arguments;
+    LOG(@"message for handle %@", message.handle);
+    AblyFlutter *const ably = [plugin ablyWithHandle:message.handle];
+    // TODO if ably is nil here then an error response, perhaps? or allow Dart side to understand null response?
+    result([ably createRestWithOptions:message.message]);
+};
+
+static FlutterHandler _publishRestMessage = ^void(AblyFlutterPlugin *const plugin, FlutterMethodCall *const call, const FlutterResult result) {
+    AblyFlutterMessage *const message = call.arguments;
+    LOG(@"message for handle %@", message.handle);
+    AblyFlutter *const ably = [plugin ablyWithHandle:message.handle];
+    AblyFlutterMessage *const messageData = message.message;
+    NSMutableDictionary<NSString *, NSObject *>* _dataMap = messageData.message;
+    NSString *channelName = (NSString*)[_dataMap objectForKey:@"channel"];
```

Comment:
Please make pointers `const` unless you have a reason for them not to be. In the case of this line that means:

```objc
NSString *const channelName = [_dataMap objectForKey:@"channel"];
```

Reply:
Handled in next PR

Reply:
Hopefully your commits are suitably granular to support illustrating in this manner but perhaps it would help if you could comment here with a link to the relevant commit so I can resolve [this thread](https://github.com/ably/ably-flutter/pull/4#discussion_r417630602).

Going forward I do want to [encourage atomic iteration on the same PR](https://github.com/ably/ably-flutter/pull/4#discussion_r419337789), but for now happy to unblock merging this PR if you can just comment in this manner here so I can resolve this comment thread. 😁

Reply:
@QuintinWillison https://github.com/ably/ably-flutter/commit/8b2817d305608163c32ae3463a35790511c73089